plugins/slack_webhooks/app/models/slack_webhook_notification.rb

```diff
@@ -17,7 +17,8 @@ def buddy_request(message)
 
   def default_buddy_request_message
     project = @deploy.project
-    ":pray: @here _#{@deploy.user.name}_ is requesting approval" \
+    # https://api.slack.com/docs/message-formatting
+    ":pray: <!here> _#{@deploy.user.name}_ is requesting approval" \
```

Comment:
can you fix it ?

On Mon, Jul 25, 2016 at 10:06 PM, Zehua Liu <notifications@github.com>
wrote:

> In plugins/slack_webhooks/app/models/slack_webhook_notification.rb
> <https://github.com/zendesk/samson/pull/1135#discussion_r72188565>:
>
> > @@ -17,7 +17,8 @@ def buddy_request(message)
> >
> >    def default_buddy_request_message
> >      project = @deploy.project
> > -    ":pray: @here _#{@deploy.user.name}_ is requesting approval" \
> > +    # https://api.slack.com/docs/message-formatting
> > +    ":pray: <!here> _#{@deploy.user.name}_ is requesting approval" \
>
> This does not seem to work on the Slack desktop client. It is showing
> verbatim <!here>. Might be due to html encoding of < >.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1135/files/b76ae95db2a8056274a2e0e81efc0275153ce103#r72188565>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ3zBqu1NTB4_6jqHRORv2cJae-p8ks5qZZXGgaJpZM4JNdff>
> .
>